pgxn/neon/extension_server.c

```diff
@@ -53,14 +53,13 @@ neon_download_extension_file_http(const char *filename, bool is_library)
     }
 
     compute_ctl_url = psprintf("http://localhost:%d/extension_server/%s%s",
-                      extension_server_port, filename, is_library?"?is_library=true":"");
+                               extension_server_port, filename, is_library ? "?is_library=true" : "");
```

Comment:
looks like unrelevant change

Reply:
my editor automatically applied style formatting to this